libs/core/src/scss/themes/_component-themes.scss

```diff
@@ -0,0 +1,17 @@
+@use '../utils';
+
+@mixin apply-dark-theme {
+  --kirby-interactive-background-color: #{utils.get-color('white-overlay')};
```

Comment:
Can we come up with a better prefix here? 
I want it to signal that this is used for our "interactive" components, like buttons, segmented control, form field etc. So it is mostly form-controls, but not quite isolated to those, as segmented control is not a form-control I guess 🤷

Reply:
That's a really good question 🤔 The only suggestion I can come up with is `--kirby-interactive-element-background-color` or `--kirby-interactive-component-background-color`. If you hadn't explained the reasoning for the naming, then I think it would have been unclear to me, what it targets 🙂 I know that the naming becomes a bit long, but I'm a firm believer in "clarity over brevity", when it comes to naming variables and functions 😄

Reply:
Agree! I changed it to `--kirby-interactive-element-background-color` and `--kirby-interactive-element-background-color`